<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmitErrorReport</class>
 <widget class="QWidget" name="SubmitErrorReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
color:black;
font-family:arial;</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2021</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="date">
    <date>
     <year>2021</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>667</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/images/reportAnIssueBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backToHome">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/backArrow2.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_reportAnError">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-size:28px;
font-family:arial;</string>
   </property>
   <property name="text">
    <string>     Report an Error</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_instructions">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:16px;
color:black;</string>
   </property>
   <property name="text">
    <string>Please be as detailed as possible to aid us in the troubleshooting process</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_subject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:16px;
color:black;</string>
   </property>
   <property name="text">
    <string>Subject:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;
border:1px solid black;
background-color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:16px;
color:black;</string>
   </property>
   <property name="text">
    <string>Date Encountered:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:16px;
color:black;</string>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Report</string>
   </property>
  </widget>
  <zorder>lbl_background</zorder>
  <zorder>dateEdit</zorder>
  <zorder>btn_backToHome</zorder>
  <zorder>lbl_reportAnError</zorder>
  <zorder>lbl_instructions</zorder>
  <zorder>lbl_subject</zorder>
  <zorder>lineEdit_title</zorder>
  <zorder>lbl_date</zorder>
  <zorder>lbl_description</zorder>
  <zorder>textEdit</zorder>
  <zorder>btn_submit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
